deploy/streisand-existing-cloud-server

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+#
+# Provision an existing cloud server.
+#
+# This requires an expanded extra-vars file specific to the provider type that
+# sets all of the values gathered by prompts in the interactive installation.
+# See the contents of global_vars/noninteractive for examples that can be copied
+# and modified.
+#
+# Usage:
+# streisand-existing-cloud-server \
+#   --provider digitalocean \
+#   --site-config path/to/digitalocean-site.yml
```

Comment:
I think this needs to also include `--ip-address` and `--ssh-user` to be a complete example for localhost noninteractive provisioning.